src/xdotoolify.js

```diff
@@ -230,7 +230,19 @@ _Xdotoolify.prototype.checkUntil = function(f, ...rest) {
   });
   return this;
 };
-_Xdotoolify.prototype.mousemove = function(selector, relpos, twoStep, timeout) {
+_Xdotoolify.prototype.checkNothing = function() {
+  this._addOperation({
+    type: 'skip'
+  });
+  return this;
+};
+_Xdotoolify.prototype.mousemove = function(
+  selector,
+  relpos,
+  twoStep,
+  timeout,
+  checkAfter = true
```

Comment:
Javascript does not have a nice way to pass named arguments. Which means that the caller will see something like `.click(selector, false)`. This does not scream UNSAFE like it should.

A better api would be `.click(selecter, {checkAfter: false})`

But I think even better would be to remove this altogether. I think it is an acceptable tradeoff that there are a few rare cases where the checkUntil is not doing anything useful in exchange for being forced to consider what every single interaction does.

Reply:
To address this I added an unsafe internal method for each of the interactions (i.e. click, type, etc.) and used this here. The default behaviour now is the safe one. To run unsafe ones, `do()` has to be run with `{unsafe: true}`.